op-conductor/flags/flags.go

```diff
@@ -91,6 +91,18 @@ var (
 		Usage:   "HTTP provider URL for execution layer",
 		EnvVars: opservice.PrefixEnvVar(EnvVarPrefix, "EXECUTION_RPC"),
 	}
+	RollupBoostRPC = &cli.StringFlag{
```

Comment:
Do we need to specify the rollup boost rpc, or will most people just be setting the execution layer rpc to point to rollup boost itself?

Reply:
Good point, I will reduce this setting to a bool flag `rollup-boost.enabled` that will refer to the value of `ExecutionRPC` for healthcheck monitoring in that case.

Reply:
Fixed in 652b78ab342cfae8635f018971efb22a756bdaba.